src/types.jl

```diff
@@ -41,15 +41,34 @@ BouncyParticle(Γ, μ, λ; ρ=0.0) = BouncyParticle(Γ, μ, λ, ρ)
     Boomerang(μ, λ) <: ContinuousDynamics
 
 Dynamics preserving the `N(μ, Σ)` measure (Boomerang)
-with refreshment time `λ`
+with refreshment time `λref`. `Γ` defines the sparse dependency graph of the target measure,
```

Comment:
Shouldn't `Γ = inv(Σ)` here?

Reply:
Right! I ll update the file today

Reply:
Actually is that right?  Γ should be the precision matrix (or the dependency graph) of the target distribution while inv(Σ) should be the precision matrix of the reference measure. Right?

Reply:
We need a dependency graph for the target, a precision for the reference measure and upper bounds. In part of the code we assume that all of this is the same. But the sparsity of the target is only used in the "fully local" algorithm. This needs to be cleaned up a bit